actgen/gscore.py

```diff
@@ -17,19 +17,12 @@ def build_confusion_matrix(q_deltas, num_duplicate):
     """
     # arrange index so similar actions are adjacent
     num_actions = len(q_deltas)
-    num_original_actions = int(num_actions / num_duplicate)
-    new_idx = []
-    for i in range(num_original_actions):
-        new_idx += list(range(i, num_actions, num_original_actions))
-    assert len(new_idx) == num_actions
-    q_deltas = q_deltas[new_idx]
```

Comment:
What's this change?

Reply:
the functionality of this block of old code is to group similar actions together in the confusion matrix. Now that we are changing the env wrapper to make similar actions already close to each other, we no longer need to do this